gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/MysqlDagStateStoreWithDagNodesTest.java

```diff
@@ -79,6 +79,8 @@ public void testAddGetAndDeleteDag() throws Exception{
     Dag<JobExecutionPlan> originalDag2 = DagTestUtils.buildDag("random_2", 456L);
     DagManager.DagId dagId1 = DagManagerUtils.generateDagId(originalDag1);
     DagManager.DagId dagId2 = DagManagerUtils.generateDagId(originalDag2);
+    originalDag1.getStartNodes().get(0).getValue().setCurrentGeneration(2);
```

Comment:
can we add one test where the dag uses the default generation and attempts (1.0)?